options.go

```diff
@@ -29,7 +29,7 @@ import (
 // Options holds Configuration Options that can be set by Command Line Flag,
 // or Config File
 type Options struct {
-	ProxyPrefix     string `flag:"proxy-prefix" cfg:"proxy-prefix" env:"OAUTH2_PROXY_PROXY_PREFIX"`
+	ProxyPrefix     string `flag:"proxy-prefix" cfg:"proxy_prefix" env:"OAUTH2_PROXY_PROXY_PREFIX"`
 	ProxyWebSockets bool   `flag:"proxy-websockets" cfg:"proxy_websockets" env:"OAUTH2_PROXY_PROXY_WEBSOCKETS"`
 	HTTPAddress     string `flag:"http-address" cfg:"http_address" env:"OAUTH2_PROXY_HTTP_ADDRESS"`
 	HTTPSAddress    string `flag:"https-address" cfg:"https_address" env:"OAUTH2_PROXY_HTTPS_ADDRESS"`
```

Comment:
`TLSCertFile` and `TLSKeyFile` are missing the `-file` part in their flags.     

(sorry comment is on the wrong line because github doesn't let comments happen in the right place)

Reply:
Very good spot, I had completely missed this, thanks! @syscll Do we change the `flag` or the `cfg` and `env`? 🤔

Reply:
My two cents are to shift to `-tls-cert-file` and `-tls-key-file` to be consistent with `-htpasswd-file` and `-jwt-key-file`. It just leaves the exception of `-logging-filename`, but it's a little bit exceptional in that it defaults to `stdout`. But I'm not strongly opinionated about which way to go.